.storybook/preview.js

```diff
@@ -27,4 +36,9 @@ export const parameters = {
     en: 'English',
     de: 'Deutsch',
   },
+  options: {
+    storySort: {
+      order: ['Getting started', 'Components'],
+    },
+  },
```

Comment:
push the helping story to the front. This is desirable anyway and helps acting as the workaround for the wrong tab selection (not docs but canvas) in the initial rendering